extension/encoding/avrologencodingextension/README.md

```diff
@@ -11,14 +11,19 @@
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
 <!-- end autogenerated section -->
 
-The `avrolog` encoding extension is used to unmarshal AVRO and insert it into the body of a log record. Marshalling is not supported.
+This extension allows to marshal and unmarshal AVRO encoded data from/into the body of a log record.
+
+The extension supports the following configuration options:
+
+- `schema_id` (optional): A schema ID used as prefix for the serialized data. If not provided or set to 0, the schema ID will not be included in the serialized data.
```

Comment:
Is this important? How do we know what to set it to?

Reply:
This was requested in the linked issue and it makes sense to support it. If kafka consumers are integrated with a schema registry for AVRO, they expect a schema id (magic byte) in front of the actual data so the schema can dynamically be fetched for the record. Configuring the collector with a static schema, one needs to know the id but helps in supporting those types of consumers.